routes/actionRouter.js

```diff
@@ -2,13 +2,26 @@ const router = require("express").Router();
 const actionDb = require("../data/helpers/actionModel.js")
 const projectDb = require("../data/helpers/projectModel.js")
 
+// get all actions for the project 
+router.get("/",(req,res) => {
```

Comment:
Okay, this should be taking in the project id and then using the getProjectActions db helper from projectDb to get the actions for the project that matches the passed in id. You should use the validateProjectId middleware. You're going to want to set the endpoint up to be something like '/projects/:id' so it doesn't conflict with your other get. I personally would actually set this up in the projects router, with something like '/:id/actions', but either way should work.